spec/controllers/series_controller_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'spec_helper'
+
+RSpec.describe SeriesController, type: :controller do
+  include LoginMacros
+  include RedirectExpectationHelper
+  let(:user) { create(:user) }
+  let(:series) { create(:series, pseuds: user.pseuds) }
+
+  describe 'new' do
+    it 'assigns a series' do
+      get :new
+      expect(assigns(:series)).to be_a_new(Series)
+    end
+  end
+
+  describe 'edit' do
+    it 'redirects to orphan if there are no psueds left.' do
+      fake_login_known_user(user)
+      get :edit, remove: "me", id: series
+      it_redirects_to(new_orphan_path(series_id: series))
+    end
+  end
+
+  describe 'create' do
+    it 'when renders new if the series is invalid' do
+      fake_login_known_user(user)
+      post :create
+      expect(response).to render_template('new')
+    end
+
+    it 'gives notice and redirects on a valid series' do
+      fake_login_known_user(user)
+      post :create, series: { title: "test title" }
+      expect(flash[:notice]).to eq "Series was successfully created."
+      expect(response).to have_http_status :redirect
+    end
+  end
+
+  describe 'update' do
+    it 'stops you removing the last author of a series' do
```

Comment:
"redirects and errors if removing the last author of a series" might be a bit more descriptive of the behavior than "stops you removing"